web/ui/templates/alerts.html

```diff
@@ -15,8 +15,11 @@ <h1>Alerts</h1>
     {{$alertStateToRowClass := .AlertStateToRowClass}}
     {{range .AlertingRules}}
       {{$activeAlerts := .ActiveAlerts}}
-      <tr class="{{index $alertStateToRowClass .State}} alert_header">
-        <td><i class="icon-chevron-down"></i> <b>{{.Name}}</b> ({{len $activeAlerts}} active)</td>
+      {{$IsHealthUnknown := .IsHealthUnknown}}
```

Comment:
To keep variable naming consistent and avoid confusion, I would suggest `$isHealthUnknown := .IsHealthUnknown`.